### DISCUSSION TITLE
change sizegaps on all workspaces

### DISCUSSION THREAD

**Discussion by andynameistaken** [asmvik/yabai#1276] [Q&A] [answered]
Hi, is it possible to increase/decrease gaps in every workspace at the same time?

**Comment by asmvik** [comment, marked as answer]
If you have not used space-specific padding, the following commands will adjust the padding on *all* spaces:

```
yabai -m config top_padding                  20
yabai -m config bottom_padding               20
yabai -m config left_padding                 20
yabai -m config right_padding                20
yabai -m config window_gap                   10
```

There is no built-in way to do an increase/decrease, but you can write a script that retrieves the current value by running `yabai -m config top_padding` and then add/subtract from the value yourself before calling `yabai -m config top_padding <value>`

**Reply by asmvik** [reply]
You can also write a script that queries all spaces `yabai -m query --spaces` and call `yabai -m space <index> --padding rel:20:0:0:0` in a for loop.

syntax is: `yabai -m space <space> --padding abs|rel:top:bottom:left:right`
